test/plugin/scenarios/graphql-9.x-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,92 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+segmentItems:
+- serviceName: graphql-9.x-scenario
+  segmentSize: gt 1
+  segments:
+  - segmentId: not null
+    spans:
+    - operationName: user
+      operationId: 0
+      parentSpanId: 0
+      spanId: 1
+      spanLayer: Unknown
+      startTime: nq 0
+      endTime: nq 0
+      componentId: 92
+      isError: false
+      spanType: Local
+      peer: ''
+      skipAnalysis: false
+      tags:
+      - {key: x-le, value: '{logic-span=true}'}
```

Comment:
Such kind of tags are not processable by the backend, according to [the doc](https://github.com/apache/skywalking/blob/310b8857b2266768ccd9e1338da13c86c313ed3f/docs/en/guides/Java-Plugin-Development-Guide.md#extension-logic-endpoint-tag-key-x-le)

> The value of x-le should be JSON format

@IanCao  did you check this plugin with the backend in a whole? FYI @wu-sheng

Reply:
Why not? I added this new feature :) It is here

https://github.com/apache/skywalking/blob/310b8857b2266768ccd9e1338da13c86c313ed3f/docs/en/guides/Java-Plugin-Development-Guide.md#extension-logic-endpoint-tag-key-x-le

Reply:
I mean `{logic-span=true}` is not JSON format, did you process this kind of format @wu-sheng ? I mean the `=`, it's not JSON format

Reply:
@wu-sheng do you mean this, I didn't see any clue that the map format `{logic-span=true}` is also supported

<img width="1202" alt="image" src="https://user-images.githubusercontent.com/15965696/86873147-5355f080-c110-11ea-8e7e-d0560c104a8a.png">

Reply:
Oh, I see, I didn't check that part.